src/app/components/patterns/MaterialDesignDescription.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { MaterialDesign } from '../../assets/icons';
+import { OpenInNew } from '@material-ui/icons';
+import {
+    Card,
+    Typography,
+    CardProps,
+    CardActionArea,
+    makeStyles,
+    createStyles,
+    Theme,
+    useTheme,
+    CardActionAreaProps,
+} from '@material-ui/core';
+
+type MaterialDesignDescriptionProps = {
+    // The icon used on the left
+    avatar?: JSX.Element;
+
+    // Secondary description text
+    description?: string;
+
+    // props applied to the card action area
+    CardActionAreaProps?: CardActionAreaProps;
+
+    // The icon used on the right
+    icon?: JSX.Element;
```

Comment:
do we expect avatar and icon to ever change? If not, we probably don't need them as props in this case.

Reply:
There are a couple spots where I think we may use this also for links to other sites, so I’d just leave the avatar there for now. 

The icons… I guess it does stay the same all the time. I am writing them almost as if they are going to be part of the component library.